fern/pages/docs/api-references/v2-parser.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: OpenAPI v2 parser
+subtitle: The next generation of the OpenAPI Parser for Docs
+---
+
+The next generation of the OpenAPI parser was built explicitly for Fern Docs. It's designed to be more expressive, more flexible and enable faster feature
```

Comment:
Features
- Parses and validates specs in JSON or YAML format
- Compatible with 2.0, OpenAPI 3.0, and AsyncAPI 3.0 schemas
- Resolves all `$ref` pointers, including external files and URLs